constants/battle_anim_constants.asm

```diff
@@ -30,687 +30,687 @@ DEF BATTLEANIM_BASE_TILE EQU 7 * 7  ; Maximum size of a pokemon picture
-	const BATTLEANIMFUNC_ANCIENT_POWER
-	const BATTLEANIMFUNC_ROCK_SMASH
-	const BATTLEANIMFUNC_COTTON
+	const BATTLE_ANIM_SEQ_NULL
+	const BATTLE_ANIM_SEQ_USER_TO_TARGET
+	const BATTLE_ANIM_SEQ_USER_TO_TARGET_DISAPPEAR
+	const BATTLE_ANIM_SEQ_MOVE_IN_CIRCLE
+	const BATTLE_ANIM_SEQ_WAVE_TO_TARGET
+	const BATTLE_ANIM_SEQ_THROW_TO_TARGET
+	const BATTLE_ANIM_SEQ_THROW_TO_TARGET_DISAPPEAR
+	const BATTLE_ANIM_SEQ_DROP
+	const BATTLE_ANIM_SEQ_USER_TO_TARGET_SPIN
+	const BATTLE_ANIM_SEQ_SHAKE
+	const BATTLE_ANIM_SEQ_FIRE_BLAST
+	const BATTLE_ANIM_SEQ_RAZOR_LEAF
+	const BATTLE_ANIM_SEQ_BUBBLE
+	const BATTLE_ANIM_SEQ_SURF
+	const BATTLE_ANIM_SEQ_SING
+	const BATTLE_ANIM_SEQ_WATER_GUN
+	const BATTLE_ANIM_SEQ_EMBER
+	const BATTLE_ANIM_SEQ_POWDER
+	const BATTLE_ANIM_SEQ_POKEBALL
+	const BATTLE_ANIM_SEQ_POKEBALL_BLOCKED
+	const BATTLE_ANIM_SEQ_RECOVER
+	const BATTLE_ANIM_SEQ_THUNDER_WAVE
+	const BATTLE_ANIM_SEQ_CLAMP_ENCORE
+	const BATTLE_ANIM_SEQ_BITE
+	const BATTLE_ANIM_SEQ_SOLAR_BEAM
+	const BATTLE_ANIM_SEQ_GUST
+	const BATTLE_ANIM_SEQ_RAZOR_WIND
+	const BATTLE_ANIM_SEQ_KICK
+	const BATTLE_ANIM_SEQ_ABSORB
+	const BATTLE_ANIM_SEQ_EGG
+	const BATTLE_ANIM_SEQ_MOVE_UP
+	const BATTLE_ANIM_SEQ_WRAP
+	const BATTLE_ANIM_SEQ_LEECH_SEED
+	const BATTLE_ANIM_SEQ_SOUND
+	const BATTLE_ANIM_SEQ_CONFUSE_RAY
+	const BATTLE_ANIM_SEQ_DIZZY
+	const BATTLE_ANIM_SEQ_AMNESIA
+	const BATTLE_ANIM_SEQ_FLOAT_UP
+	const BATTLE_ANIM_SEQ_DIG
+	const BATTLE_ANIM_SEQ_STRING
+	const BATTLE_ANIM_SEQ_PARALYZED
+	const BATTLE_ANIM_SEQ_SPIRAL_DESCENT
+	const BATTLE_ANIM_SEQ_POISON_GAS
+	const BATTLE_ANIM_SEQ_HORN
+	const BATTLE_ANIM_SEQ_NEEDLE
+	const BATTLE_ANIM_SEQ_PETAL_DANCE
+	const BATTLE_ANIM_SEQ_THIEF_PAYDAY
+	const BATTLE_ANIM_SEQ_ABSORB_CIRCLE
+	const BATTLE_ANIM_SEQ_BONEMERANG
+	const BATTLE_ANIM_SEQ_SHINY
+	const BATTLE_ANIM_SEQ_SKY_ATTACK
+	const BATTLE_ANIM_SEQ_GROWTH_SWORDS_DANCE
+	const BATTLE_ANIM_SEQ_SMOKE_FLAME_WHEEL
+	const BATTLE_ANIM_SEQ_PRESENT_SMOKESCREEN
+	const BATTLE_ANIM_SEQ_STRENGTH_SEISMIC_TOSS
+	const BATTLE_ANIM_SEQ_SPEED_LINE
+	const BATTLE_ANIM_SEQ_SLUDGE
+	const BATTLE_ANIM_SEQ_METRONOME_HAND
+	const BATTLE_ANIM_SEQ_METRONOME_SPARKLE_SKETCH
+	const BATTLE_ANIM_SEQ_AGILITY
+	const BATTLE_ANIM_SEQ_SACRED_FIRE
+	const BATTLE_ANIM_SEQ_SAFEGUARD_PROTECT
+	const BATTLE_ANIM_SEQ_LOCK_ON_MIND_READER
+	const BATTLE_ANIM_SEQ_SPIKES
+	const BATTLE_ANIM_SEQ_HEAL_BELL_NOTES
+	const BATTLE_ANIM_SEQ_BATON_PASS
+	const BATTLE_ANIM_SEQ_CONVERSION
+	const BATTLE_ANIM_SEQ_ENCORE_BELLY_DRUM
+	const BATTLE_ANIM_SEQ_SWAGGER_MORNING_SUN
+	const BATTLE_ANIM_SEQ_HIDDEN_POWER
+	const BATTLE_ANIM_SEQ_CURSE
+	const BATTLE_ANIM_SEQ_PERISH_SONG
+	const BATTLE_ANIM_SEQ_RAPID_SPIN
+	const BATTLE_ANIM_SEQ_BETA_PURSUIT
+	const BATTLE_ANIM_SEQ_RAIN_SANDSTORM
+	const BATTLE_ANIM_SEQ_BATTLE_ANIM_INDEX_B0
+	const BATTLE_ANIM_SEQ_PSYCH_UP
+	const BATTLE_ANIM_SEQ_ANCIENT_POWER
+	const BATTLE_ANIM_SEQ_ROCK_SMASH
+	const BATTLE_ANIM_SEQ_COTTON
 DEF NUM_BATTLEANIMFUNCS EQU const_value
```

Comment:
```suggestion
DEF NUM_BATTLE_ANIM_SEQS EQU const_value
```

Reply:
Actually, should we consistently call these "sprite/battle anim funcs" instead of "sprite/battle anim seqs"?

Reply:
I didn't really have a preference either way; I just chose sequence for the meantime. I'm fine with changing them to funcs instead. I'll work on that tonight.